doc/source/skips/2-values.rst

```diff
@@ -0,0 +1,178 @@
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
+old contributors coming in, adding what small bits they need at the time, and
+disappearing again for a while. This is *fine* and we want to encourage more of
+it, but it also lacks direction. Additionally, without a concerted roadmap to
+concentrate effort, many of these contributions just fall by the wayside, as it
+is difficult for new contributors to live up to our stringent (and largely
+unwritten) standards of code.
+
+Implementation
+--------------
+
+Our mission
+***********
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new
+  dependencies, and sometimes cull existing ones, or make them optional. All
+  functions in our API have thorough docstrings clarifying expected inputs and
+  outputs.
+- providing a **consistent API**. Conceptually identical arguments have the
+  same name and position in a function signature.
+- **ensuring accuracy**. Test coverage is close to 100% and code is reviewed by
+  at least two core developers before being included in the library.
+- **caring for users’ data**. We have a functional [2]_ API and don't modify
+  input arrays unless explicitly directed to do so.
+- promoting **education in image processing** with extensive pedagogical
+  documentation.
+
+Our values
+**********
+
+- We are inclusive. We continue to welcome and mentor newcomers who are
+  making their first contribution.
+- We are community-driven. Decisions about the API and features are driven by
+  our users' requirements, not by the whims of the core team. (See
+  :ref:`governance`.)
+- We serve scientific applications primarily, over “consumer” image editing in
+  the vein of Photoshop or GIMP. This often means prioritizing n-dimensional
+  data support, and rejecting implementations of “flashy” filters that have
+  little scientific value.
+- We value simple, readable implementations over getting every last ounce of
+  performance. Readable code that is easy to understand, for newcomers and
+  maintainers alike, makes it easier to contribute new code as well as prevent
+  bugs. This means that we will prefer a 20% slowdown if it reduces lines of
+  code two-fold, for example.
+- We value education and documentation. All functions should have NumPy-style
+  docstrings [3]_, preferably with examples, as well as gallery
+  examples that showcase how that function is used in a scientific application.
+  Core developers take an active role in finishing documentation examples.
+- We don't do magic. We use NumPy arrays instead of fancy façade objects
+  [#np]_, and we prefer to educate users rather than make decisions on their
+  behalf.  This does not preclude sensible defaults, but it does preclude
+  *value-dependent* behavior that is not controllable by the user. [4]_
```

Comment:
I think, without relevant contex, most people won't get what "value dependent" behavior means.

Reply:
I'm not sure how to make this more concise though...

Reply:
Maybe just remove this sentence?

Reply:
I'll leave it at "this does not preclude sensible defaults". However, I think somewhere in our reviewing guidelines we should write that value-dependent API behavior changes should be avoided, as it's pretty typical and tempting to do in Python.